magefile.go

```diff
@@ -2372,6 +2376,32 @@ func (Integration) TestKubernetesMatrix(ctx context.Context) error {
 	return integRunner(ctx, "testing/integration/k8s", true, "")
 }
 
+// BuildKubernetesTestData builds the test data required to run k8s integration tests
+func (Integration) BuildKubernetesTestData(ctx context.Context) error {
+	// build the dependencies for the elastic-agent helm chart
+	mg.Deps(Helm.BuildDependencies)
+
+	// download opentelemetry-kube-stack helm chart
+	kubeStackHelmChartPath, err := devtools.DownloadFile(k8s.KubeStackChartURL, filepath.Join("testing", "integration", "k8s", "testdata"))
+	if err != nil {
+		return fmt.Errorf("failed to download opentelemetry-kube-stack helm chart: %w", err)
+	}
+	if err := os.Rename(kubeStackHelmChartPath, filepath.Join("testing", "integration", "k8s", k8s.KubeStackChartPath)); err != nil {
```

Comment:
Why is this os.Rename needed? From the code it seems to me like the `DownloadFile` function above should create the file with the correct name in the correct position :thinking:

Reply:
yep you are right, I guess it's just me, but I would like the `devtools.DownloadFile` to accept a targetPath and not a targetDir to feel "safer" 😄  pushed this here 1196468261caa53561a80512368327bd58e3753d